src/features/favorites/api/useAddFavorite.test.tsx

```diff
@@ -15,7 +14,15 @@ import { env } from 'libs/environment'
 import { EmptyResponse } from 'libs/fetch'
 import { reactQueryProviderHOC } from 'tests/reactQueryProviderHOC'
 import { server } from 'tests/server'
-import { renderHook, superFlushWithAct } from 'tests/utils'
+import { waitFor } from 'tests/utils'
+import { renderHook } from 'tests/utils'
```

Comment:
Import dupliqué ici : 

```suggestion
import { waitFor, renderHook } from 'tests/utils'
```